scripts/index.js

```diff
@@ -0,0 +1,117 @@
+// //color change function
+
+// const changeColorTempDisplay = function(val) {
+//     let colorThreshold = 20;
+//     let tempValue = document.getElementById('tempDisplay');
+//     console.log(tempValue)
+
+//     const red = '#f60707'
+//     const orange = '#FF5733'
+//     const yellow = '#FFC300'
+//     const green = '#19990c'
+//     const blue = '#008080'
+
+//     if (tempValue <= 49) {
+//         tempValue.style.color = blue;
+//     } else if (tempValue >= 50) {
+//         tempValue.style.color = green;
+//     }
+// }
+
+//change landscapes
+
+// const weatherEmoji = document.createElement("span")
+// const weatherContainer = document.querySelector("#weatherWindow")
+// weatherEmoji.textContent = ❄️;
+// weatherContainer.appendChild(weatherEmoji)
+
+//increase temp by one click
+
+
+const state = {
+    tempDisplay: 0,
```

Comment:
This should probably match the initial state of the HTML.

```suggestion
    tempDisplay: 20,
```